lightning/src/ln/channel.rs

```diff
@@ -1379,13 +1379,15 @@ impl<Signer: Sign> Channel<Signer> {
 					_ => {}
 				}
 			}
+			log_trace!(logger, "Placing failure for HTLC ID {} in holding cell", htlc_id_arg);
 			self.holding_cell_htlc_updates.push(HTLCUpdateAwaitingACK::FailHTLC {
 				htlc_id: htlc_id_arg,
 				err_packet,
 			});
 			return Ok(None);
 		}
 
+		log_trace!(logger, "Failure HTLC ID {} back with a update_fail_htlc message", htlc_id_arg);
```

Comment:
Can't quite parse this log message... s/Failure/Failing?

Reply:
Oops, yes, should be "Failing".